jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/JsonExceptionMapper.java

```diff
@@ -52,9 +52,13 @@
     public final Response toResponse(T exception) {
         String exceptionMessage = Objects.toString(exception.getMessage());
         String errorId = UUID.randomUUID().toString();
-        log.error("Error handling request {}", errorId, exception);
-
         StatusType status = this.getStatus(exception);
+        if (status.getFamily().equals(Response.Status.Family.CLIENT_ERROR)) {
+            log.info("Error handling request {}", errorId, exception);
+        } else {
+            log.error("Error handling request {}", errorId, exception);
```

Comment:
yeah, I think we probably need to make a more concerted effort internally on adoption of SafeLoggable.